src/main/scala/ru/d4team/chat/config/PostgresConfig.scala

```diff
@@ -1,28 +1,47 @@
 package ru.d4team.chat.config
 
-import zio._
+import org.flywaydb.core.Flyway
+import org.flywaydb.core.api.configuration.FluentConfiguration
 import zio.sql.HikariConnectionPoolConfig
+import zio.{ RLayer, ZIO, ZLayer }
 
 final case class PostgresConfig(
-    host: String,
-    port: Int,
-    db: String,
-    user: String,
-    password: String
+  host: String,
+  port: Int,
+  db: String,
+  user: String,
+  password: String,
+  migrationsLocation: String,
+  schema: String
 ) {
   val url = s"jdbc:postgresql://$host:$port/$db"
 }
 
 object PostgresConfig {
 
+  val flywayConfig: RLayer[PostgresConfig, FluentConfiguration] = ZLayer.fromFunction(buildFlywayConfig _)
+  private def buildFlywayConfig(conf: PostgresConfig): FluentConfiguration =
+    Flyway.configure
+      .loggers("slf4j")
+      .dataSource(
+        conf.url,
+        conf.user,
+        conf.password
+      )
+      .group(true)
+      .outOfOrder(false)
+      .locations(conf.migrationsLocation)
+      .failOnMissingLocations(true)
+      .baselineOnMigrate(true)
+
```

Comment:
We can simplify this to:
``` scala
val flywayConfig: RLayer[PostgresConfig, FluentConfiguration] =
  ZLayer.fromFunction { conf: PostgresConfig =>
    Flyway.configure
      .loggers("slf4j")
      .dataSource(conf.url, conf.user, conf.password)
      .group(true)
      .outOfOrder(false)
      .locations(conf.migrationsLocation)
      .failOnMissingLocations(true)
      .baselineOnMigrate(true)
  }
```

Reply:
yep, that looks better)
Thank you
corrected